var/spack/repos/builtin/packages/fenics-dolfinx/package.py

```diff
@@ -15,42 +15,45 @@ class FenicsDolfinx(CMakePackage):
     maintainers = ["chrisrichardson", "garth-wells", "nate-sime"]
 
     version("main", branch="main")
+    version("0.4.1", sha256="68dcf29a26c750fcea5e02d8d58411e3b054313c3bf6fcbc1d0f08dd2851117f")
     version("0.3.0", sha256="4857d0fcb44a4e9bf9eb298ba5377abdee17a7ad0327448bdd06cce73d109bed")
     version("0.2.0", sha256="4c9b5a5c7ef33882c99299c9b4d98469fb7aa470a37a91bc5be3bb2fc5b863a4")
     version("0.1.0", sha256="0269379769b5b6d4d1864ded64402ecaea08054c2a5793c8685ea15a59af5e33")
 
     variant("kahip", default=False, description="kahip support")
-    variant("parmetis", default=False, description="parmetis support")
     variant("slepc", default=False, description="slepc support")
     variant("adios2", default=False, description="adios2 support")
+    variant("scotch", default=False, description="scotch support")
```

Comment:
I think when `scotch` is activated, we need `scotch_mpi`.

Reply:
I thought scotch mpi support would be obtained with the `depends_on("scotch+mpi", when"+scotch")` line 42.